app/src/main/res/values/strings.xml

```diff
@@ -225,6 +225,10 @@
   <string name="add_profile_error_name_only_letters">Names can only have letters. Try another name?</string>
   <string name="add_profile_error_pin_length">Your PIN should be 3 digits long.</string>
   <string name="add_profile_error_pin_confirm_wrong">Please make sure that both PINs match.</string>
+  <string name="add_profile_asterisk_required">Fields marked with an * are required.</string>
```

Comment:
Rename these strings to following:
`required_fields_content_description`
`current_profile_picture_content_description`
`edit_profile_picture_content_description`
`add_profile_info_content_description`

Reply:
I have renamed as per convention. Please review.

> @yashraj-iitr The strings are still not correct.
> The reason why these strings do not contain `add_profile` as prefix is because they are general strings and can be used at other places in code too
> [#2653 (comment)](https://github.com/oppia/oppia-android/pull/2653#discussion_r572078863)

okay. I get it now.

Reply:
@rt4914 I believe now it is correct. Please have a look.